programmers/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,6 @@
+function solution(s) {
+    return s.length % 2 ?
+        s.slice(s.length / 2, s.length / 2 + 0.5)
+        :
+        s.slice(s.length / 2 - 1, s.length / 2 + 1);
+}
```

Comment:
```suggestion
function solution(s) {
    const even = s.length / 2;
    const odd = Math.floor(s.length / 2);
    
    return s.length % 2 ? s[odd] : s[even - 1] + s[even]
}
```
일단 너무 잘 풀었어요 
이 문제는 참고만하셔도됩니다 ~